jobs/Backend/Task/ExchangeRateUpdater/appsettings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "BaseUrl": "https://api.cnb.cz/"
```

Comment:
Nice practice to put the URL into a separated settings file - would be easier to change it later also by non-technical people in the future 👍 (fellow candidate here)